### DISCUSSION TITLE
[More Effective C#] Chapter 3. 요약 (1)

### DISCUSSION THREAD

**Discussion by hyun98** [BDD-CLUB/challenge-every-week-posting#410] [3. CS(컴퓨터 공학) 관련 글 모음]
Chapter 3의 Item 27 ~ 30 을 요약해봤어요
비동기 프로그래밍 어렵네요
[[More Effective C#] Chapter 3. 요약 (1)](https://hyeo-noo.tistory.com/425)

**Comment by 02ggang9** [comment]
비동기 글 잘 봤습니다..!! 요즘 비동기 글 보이면 너무 재미있네요..ㅎㅎ
